includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -434,43 +474,86 @@ protected function setup_assets() {
 			new Script(
 				'googlesitekit-modules-reader-revenue-manager',
 				array(
-					'src'          => $base_url . 'js/googlesitekit-modules-reader-revenue-manager.js',
-					'dependencies' => array(
+					'src'           => $base_url . 'js/googlesitekit-modules-reader-revenue-manager.js',
+					'dependencies'  => array(
 						'googlesitekit-vendor',
 						'googlesitekit-api',
 						'googlesitekit-data',
 						'googlesitekit-modules',
+						'googlesitekit-notifications',
 						'googlesitekit-datastore-site',
 						'googlesitekit-datastore-user',
 						'googlesitekit-components',
 					),
+					'load_contexts' => array( Asset::CONTEXT_ADMIN_SITEKIT, Asset::CONTEXT_ADMIN_POST_EDITOR ),
```

Comment:
Questions for clarification: Is adding the `CONTEXT_ADMIN_POST_EDITOR` context here necessary? Shouldn't adding this asset as a dependency for block editor assets be sufficient? Won't this change make this asset load in the block editor even when it is not required?

Reply:
Good spot! You're right - the asset will still be registered in the post editor screen without specifying the load context, and we can rely on the dependency chain to ensure it's enqueued and avoid it being enqueued when it's not needed.

I've removed this unneeded `load_contexts` definition.